src/js/claims-status/containers/AppealInfo.jsx

```diff
@@ -1,20 +1,69 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Link } from 'react-router';
+
+import moment from 'moment';
+
+import Breadcrumbs from '../components/Breadcrumbs';
 import AppealsV2TabNav from '../components/appeals-v2/AppealsV2TabNav';
 
-const AppealInfo = ({ params, children }) => {
-  const appealId = params.id;
-  return (
-    <div>
-      <AppealsV2TabNav appealId={appealId}/>
-      {children}
-    </div>
-  );
-};
+function isolateAppeal(state, id) {
+  const claimsState = state.disability.status;
+  const { appeals } = claimsState.claims;
+  const appeal = appeals.find(a => a.id === id);
+
+  // append starting event for post-remand and post-cavc remand appeals
+  // NOTE: This is business logic pulled from v1 that we don't fully understand yet.
+  //  Also, having this logic in mapStateToProps is less than ideal; we want to
+  //  move it out when we know where it should live. Maybe just a helper.
+  if (appeal && appeal.attributes.type !== 'original' && appeal.attributes.prior_decision_date) {
+    appeal.attributes.events.unshift({
+      type: appeal.attributes.type === 'post_cavc_remand' ? 'cavc_decision' : 'bva_remand',
+      date: appeal.attributes.prior_decision_date,
+    });
+  }
+
+  return appeal;
+}
+
+class AppealInfo extends React.Component {
+  render() {
+    const { params, children } = this.props;
+    const appealId = params.id;
+    // TODO: Make sure this is where we should get the date from
+    const appealDate = this.props.appeal ? moment(this.props.appeal.attributes.events[0].date, 'YYYY-MM-DD').format('MMMM YYYY') : '';
```

Comment:
@cmgiven Should the date that shows in the breadcrumbs come from the first event in the array?

Reply:
The first event with type `claim`. We should also be able to handle the absence of such an event, in which case it might just read "Appeal of Claim Decision".

cc @gnakm

Reply:
I was thinking something like Appeal of Claim Decision [Month Year]. This is something I want to go over with Amy Behrens when she comes town. 

![image](https://user-images.githubusercontent.com/13420618/32955819-119f58b2-cb85-11e7-95f4-a75b4f962a2c.png)

Reply:
Maybe stick with Appeal of Claim Decision.

Reply:
@cmgiven Excellent. I can make that filter and leave off the date if there are no events of type `claim`. 👍